tests/integration/resin.spec.coffee

```diff
@@ -101,3 +101,20 @@ describe 'Resin SDK', ->
 					m.chai.expect(called).to.equal true,
 						'responseError should be called when authentication fails'
 
+		describe 'getSdk.setSharedOptions()', ->
```

Comment:
Really these should be unindented a level - they're currently within `factory function`, but it's not calling the factory function (directly at least)